static/locales/de.json

```diff
@@ -8,6 +8,125 @@
   "dark_style_nl": "Dunkel (Keine Labels)",
   "style_light2_nl": "Licht2 (Keine Labels)",
   "style_pgo_nl": "PokemonGo (Keine Labels)",
+  "Options":"Einstellungen",
+  "Marker Settings":"Symbol Einstellungen",
+  "Stats":"Statistik",
+  "Gym/Raid":"Arenen/Raids",
+  "Only Active Raids":"Nur aktive Raids",
+  "Minimum Raid Level":"Minimum Level der Raids",
+  "Maximum Raid Level":"Maximum Level der Raids",
+  "Use Gym Sidebar":"Seitenleiste für Arenen",
+  "Gyms":"Arenen",
+  "Team":"Team",
+  "All":"Alle",
+  "None":"Keine",
+  "Uncontested":"Unbesetzt",
+  "Mystic":"Weisheit",
+  "Valor":"Wagemut",
+  "Instinct":"Intuition",
+  "Open Spot":"Freie Plätze",
+  "Minimum Free Slots":"Mindestens freie Plätze",
+  "Maximum Free Slots":"Maximal freie Plätze",
+  "Last Scan":"Letzter Scan",
+  "Last Hour":"Letzte Stunde",
+  "Last 6 Hours":"Letzten 6 Stunden",
+  "Last 12 Hours":"Letzten 12 Stunden",
+  "Last 24 Hours":"Letzten 24 Stunden",
+  "Last Week":"Letzte Woche",
+  "Only Lured":"Nur mit Lockmodul aktive",
+  "Scanned Locations":"Gescannte Bereiche",
+  "Spawn Points":"Spawn Punkte",
+  "Spawn Point":"Spawn Punkt",
+  "Every hour from":"Jede Stunde von",
+  "to":"bis",
+  "May appear as early as":"Erscheint vermutlich frühestens um",
+  "Ranges":"Reichweiten",
+  "Hide Pokemon":"Ausgeblendete",
```

Comment:
Würde ich in "Pokemon ausblenden" umbennen

Reply:
passt leider dann nicht mehr in das Tab.
Die jetzige Anzahl (13 chars) passt gerade noch so.
(das Tab ist unter Pokemon, bei den beiden Tabs zu auswahl)

Reply:
Ok, das ist natürlich ein Argument ;)